src/Game.js

```diff
@@ -47,6 +47,10 @@ class Game
 		{
 			document.getElementById("message").innerHTML = "The game is now over!";
 		}
+		if (this.gameState === 0 && this.nextTurn.player.isCPU)
+		{
+			this.Step(this.nextTurn.player.GetNextMove(this.board, this));
```

Comment:
Update so this can run in a loop. That is, so 2 CPUs can play against each other without a slowly but inexorably increasing call stack.